Dev

EDyugaeva opened the pull request:
to send homework
added:
controllers
cat shelter (menu)
button back

EDyugaeva committed:
repositories
some change in model, now app is working

EDyugaeva committed:
Merge branch 'dev' into repository

EDyugaeva committed:
Saving reports to database
Saving photo has to be corrected
Saving text works correctly

EDyugaeva committed:
Update Pet.java

EDyugaeva committed:
Update PetOwner.java

EDyugaeva committed:
Merge pull request #11 from EDyugaeva/repository

repositories

EDyugaeva committed:
Merge branch 'dev' into Branch_Kate

EDyugaeva committed:
Merge pull request #13 from EDyugaeva/Branch_Kate

Saving reports to database

EDyugaeva committed:
find some mistakes

EDyugaeva committed:
mistakes

EDyugaeva committed:
Merge pull request #14 from EDyugaeva/repository

find some mistakes in repository

SergeFox02 committed:
UpdateProject

SergeFox02 committed:
Update All Project

EDyugaeva committed:
Update migrations.sql

SergeFox02 committed:
Small Refactor code

SergeFox02 committed:
Merge remote-tracking branch 'origin/dev' into SergeBranch

# Conflicts:
#	src/main/java/pro/sky/animal_shelter_telegram_bot/repository/ReportRepository.java
#	src/main/java/pro/sky/animal_shelter_telegram_bot/service/impl/PhotoOfPetServiceImpl.java
#	src/main/java/pro/sky/animal_shelter_telegram_bot/service/impl/VolunteerServiceImpl.java
#	src/main/resources/liquibase/scripts/migrations.sql

EDyugaeva committed:
Create "Controller Advisor"
Error Handling for REST with Spring

SergeFox02 committed:
Small Refactor code

EDyugaeva committed:
Create "Button Back"

SergeFox02 committed:
Corrected errors according to the comments of the mentor

SergeFox02 committed:
Add in gitignore: *.iml and /.idea/

SergeFox02 committed:
Add in gitignore: *.iml and /.idea/

SergeFox02 committed:
Merge remote-tracking branch 'origin/SergeBranch' into dev

EDyugaeva committed:
Update .gitignore

gitigniore

EDyugaeva committed:
Merge pull request #18 from EDyugaeva/create_button_back

Create "Button Back"

EDyugaeva committed:
Added some function to cat shelter (only in menu)

SergeFox02 committed:
Update PetOwnerController, PetOwnerService

SergeFox02 committed:
Update ReportController, ReportService

SergeFox02 committed:
Update VolunteerController, VolunteerService

SergeFox02 committed:
Update PhotoOfPetController

SergeFox02 committed:
Merge pull request #22 from EDyugaeva/SergeBranch

Update Controllers (Serge)

EDyugaeva committed:
Commit with changed saving phone numbers (sharing contacts)

EDyugaeva committed:
Merge pull request #21 from EDyugaeva/schelter_to_cats

Added some function to cat shelter (only in menu)

EDyugaeva committed:
Merge branch 'dev' into phone_number

EDyugaeva committed:
Saving PhotoOfPet

OlgaLugacheva commented on src/main/java/pro/sky/animal_shelter_telegram_bot/service/impl/PetOwnerServiceImpl.java:
```diff
@@ -37,23 +37,35 @@ public void deletePetOwner(PetOwner petOwner) {
     }
 
     @Override
     public PetOwner findPetOwner(Long id) {
```
тут лучше бы возвращать Optional а обработку перенести в контроллер
 ато получается два раза проверка

EDyugaeva committed:
Merge pull request #23 from EDyugaeva/phone_number

Commit with changed saving phone numbers (sharing contacts)

EDyugaeva committed:
Update TelegramBotUpdatesListener.java

mistake

EDyugaeva committed:
Merge pull request #24 from EDyugaeva/sharing_photos

Sharing photos

EDyugaeva committed:
Update TelegramBotUpdatesListener.java

mistake

EDyugaeva committed:
Merge pull request #15 from EDyugaeva/exceptions

Create "Controller Advisor"

EDyugaeva committed:
Calling volunteer

Saving phone number in controller (and setting chat id)

EDyugaeva committed:
Update pom.xml

added dependency

EDyugaeva committed:
Merge pull request #26 from EDyugaeva/ask_volunteer

Calling volunteer

avvasil committed:
Scheduler Service added

EDyugaeva committed:
Calling volunteer

Saving phone number in controller (and setting chat id)

SergeFox02 committed:
Corrected method  deletePet in PetService, PetController

SergeFox02 committed:
Corrected method  deletePetOwner in PetOwnerService, PetOwnerController

SergeFox02 committed:
Corrected method deleteReport in ReportService, ReportController

SergeFox02 committed:
Corrected method deleteVolunteer in VolunteerService, VolunteerController

SergeFox02 committed:
Update documentation in method findAll in VolunteerController

EDyugaeva committed:
Merge branch 'dev' into Alexandr_branch

EDyugaeva committed:
Merge pull request #27 from EDyugaeva/Alexandr_branch

Scheduler Service added
(resolved conflicts, next will check how correct it was)

EDyugaeva committed:
Resolved conflicts from Alexandr branch

EDyugaeva committed:
Merge pull request #28 from EDyugaeva/ask_volunteer

Calling volunteer

EDyugaeva committed:
descriptions and fixing bags in asking volunteer (if the list of volunteer is empty, then send a message "they are busy")

SergeFox02 committed:
Merge remote-tracking branch 'origin/SergeBranch' into dev

SergeFox02 committed:
Corrected PhotoController is work getPhoto

EDyugaeva committed:
Ending of probation
Getting photo from swagger (by report id) fixed

I do not now how to set media type, so I deleted it
